readme.md

```diff
@@ -149,43 +115,46 @@ that you created in **Prerequisites**.
     Where you replace `<your-project-id>`  with the identifier of your cloud
     project.
 
-1. Include the Python API client in your App Engine application.
+1.  Create the Pub/Sub topic that you will push messages to.
 
-        $ pip install -t gae/lib/ google-api-python-client
+        $ gcloud pubsub topics create test
 
-    Note: if you get an error and used Homebrew to install Python on OS X,
-    see [this fix](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Homebrew-and-Python.md#note-on-pip-install---user).
+The topic is now listed under `gcloud pubsub topics list`.  You can also see the topic
+(after about a minute) in the console:
```

Comment:
I think this create is now much faster than 1-minute

Reply:
Ta, removed.